src/RefractAPI.cc

```diff
@@ -39,6 +41,18 @@ namespace drafter {
             }
         }
 
+        refract::ArrayElement* CreateArrayElement(refract::IElement* value, bool rFull = false)
```

Comment:
We already have a `CreateArrayElement` in this file. Please use that by adding a parameter.